components/google-cloud/dependencies.py

```diff
@@ -16,8 +16,9 @@
 
 def make_required_install_packages():
     return [
-        "kfp>=1.4.0,<2.0.0",
+        "google-api-core<2dev,>=1.21.0",
```

Comment:
I think this can be bumped to `>=1.22.2`: https://github.com/googleapis/python-aiplatform/blob/v1.1.1/setup.py#L65

Unless we also want to bump google-cloud-aiplatform to `1.3` which can tighten the bound to `>=1.26.0`: https://github.com/googleapis/python-aiplatform/blob/v1.3.0/setup.py#L71

Reply:
Done. Good call, I think we should bump both.